### ISSUE TITLE
Include the domain models and API endpoints

### ISSUE THREAD

**Issue by chenrui333** [Brightspace/valence-sdk-java#8]
It would be nice to include each domain models and API endpoints for application integration.

**Comment by jheadley** [comment]
I agree. I would also add that publishing to a public maven repo so that users don't have to build and publish themselves would be useful.
